Package/Captcha/Captcha.php

```diff
@@ -410,6 +428,28 @@ public function setSize($size)
     }
 
     /**
+     * Set widget language code
+     * @param string $languageCode
+     * @return Captcha
+     * @see https://developers.google.com/recaptcha/docs/language
+     */
+    public function setLanguageCode($languageCode)
+    {
+        $this->languageCode = $languageCode;
+        return $this;
+    }
+    
+    /**
+     * Get widget language code
+     * @return string
+     * @see https://developers.google.com/recaptcha/docs/language
```

Comment:
Please move @see above @return for consistency sake. There should be a space between description and phpDoc blocks. Appreciate it.